CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 3.1.3
+
+### Bug Fixes
+
+- [#395](https://github.com/okta/okta-auth-js/pull/395) Prevents concurrent use of token API methods such as `getWithoutPrompt`, `getWithRedirect` or `getWithPopup` within a single running instance. These methods will be executed within a queue to ensure that they complete sequentially. This fix only affects a single instance. If there are several instances running (for example, in multiple tabs) it is still possible for token API methods to be executing concurrently.
```

Comment:
Doesn't multiple-tabs get fixed by the switch to sessionStorage in 3.1.2?  Or have you found another case where it is still an issue?

Reply:
There is still validation of the OAUTH_PARAMS cookie, that can break with multiple tabs. We need to switch this to session storage, but provide an option to use cookies for server-side apps. This would be a breaking change though.